tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckCLDR.java

```diff
@@ -196,12 +204,14 @@ public StatusAction getShowRowAction(
 
             // if limited submission, and winner doesn't have an error, limit the values
 
-            if (LIMITED_SUBMISSION && !SubmissionLocales.allowEvenIfLimited(pathValueInfo.getLocale().toString(), pathValueInfo.getXpath(), valueStatus == ValueStatus.ERROR, pathValueInfo.getBaselineStatus() == Status.missing)) {
-                return StatusAction.FORBID_READONLY;
+            if (LIMITED_SUBMISSION) {
+                if (!SubmissionLocales.allowEvenIfLimited(pathValueInfo.getLocale().toString(), pathValueInfo.getXpath(), valueStatus == ValueStatus.ERROR, pathValueInfo.getBaselineStatus() == Status.missing)) {
+                    return StatusAction.FORBID_READONLY;
+                }
```

Comment:
This change simply prevents Eclipse from warning about dead code when LIMITED_SUBMISSION is false